src/clusterfuzz/_internal/fuzzing/corpus_manager.py

```diff
@@ -243,6 +250,19 @@ def get_gcs_url(self, suffix=''):
 
     return url
 
+  def get_zipcorpus_gcs_url(self, suffix=''):
+    """Build zipcorpus GCS URL for gsutil.
+    Returns:
+      A string giving the GCS URL.
+    """
+    url = f'gs://zc/{self.bucket_name}{self.bucket_path}{suffix}'
```

Comment:
I need to add something to distinguish from the ordinary corpus. I suppose I could make some kind of change later that drops this while preserving corpora (e.g. we read from both /zc/ and without it).